test/dish_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+
+class DishTest < Minitest::Test 
+
+  def test_it_exists
+    dish = Dish.new(name, :appetizer) 
+    
+    assert_instance_of Dish, dish 
+  end 
+  
+  def test_it_has_a_dish_name
+    dish = Dish.new("Couscous Salad", :appetizer)
+    
+    assert_equal "Couscous Salad", dish.name 
+  end 
+  
+  def test_it_has_a_dish_category
```

Comment:
Test more than one dish type!  Right now this works because the way you have your Dish class set up, _every_ dish will be an appetizer.